### PR TITLE
feat: add ratings and toast component

### PR DESCRIPTION
**Pull request by rahulbakshi4**
Feature Added:
1. Ratings
2. Toasts

### CODE REVIEW

File: Components/Ratings/rating.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.rated {
+    color: rgb(255, 227, 0);
```
Comment (OmKakatkar) [inline Components/Ratings/rating.css]: Try to use CSS variables instead of hardcoding the color

File: Components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    color: #009485;
+    background-color: #e5f4f3;
+  }
+  .toast-error {
+    border-top: 1px solid #ff5724;
+    border-right: 1px solid #ff5724;
+    border-bottom: 1px solid #ff5724;
+    border-left: 4px solid #ff5724;
+    color: #ff5724;
+    background-color: #ffeee9;
```
Comment (OmKakatkar) [inline Components/Toast/toast.css]: Here as well, try using variables for colors as you can change them later if required.
Also, you can also declare a `border` property for all sides first and then re-style the `border-left-width` property. May save some repeated code.

**Review by OmKakatkar — CHANGES_REQUESTED** [review]
The UI looks clean!

File: Components/Toast/toast.css (hunk 2 above)
Comment (rahulbakshi4) [reply Components/Toast/toast.css]: Oh thats my bad I forget to used the variables I declared. Thanks for this

Comment (rahulbakshi4) [reply Components/Toast/toast.css]: Pushed the changes !!

**Review by OmKakatkar — APPROVED** [review]
Looks good!

**Review by shrey27 — APPROVED** [review]

**Pull request merged**